### DISCUSSION TITLE
Terazi denge sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#37] [Zeka Sorusu]
Terazinin bir kefesinde 200 gr, diğer kefesine 100 gr gelen içi su dolu bardak ve 100 gr ağırlık koyuyorsunuz. Terazi dengededir.
Şimdi 100 gr’lık ağırlığı su dolu bardağın içine atıyorsunuz, su taşmıyor tabi fakat yükseliyor.
Denge bozulur mu?

**Comment by FatihOnder81** [comment]
Hayır bozulmaz çünkü bir tarafta 200gr var diğer tarfta 100gr su dolu bardak ve 100gr ağırlık var  100gr ağırlığı suyun içine atarsak o bardağın toplam ağırlığı 200gr olur ve dengede olur

**Comment by yukseel** [comment]
Bozulmaz, hacim artar fakat ağırlık değişmez
